etc/rules/0570-policy_rules.xml

```diff
@@ -10,42 +10,79 @@
+  </rule>
+
+  <rule id="19004" level="7">
+    <if_sid>19001</if_sid>
+    <field name="configuration_assessment.score">^4\d|^3\d</field>
+    <description>Configuration Assessment summary: $(configuration_assessment.name): Score less than 50% ($(configuration_assessment.score))</description>
   </rule>
 
-  <rule id="19002" level="0">
+  <rule id="19005" level="9">
+    <if_sid>19001</if_sid>
+    <field name="configuration_assessment.score">^2\d|^1\d|^\d$</field>
+    <description>Configuration Assessment summary: $(configuration_assessment.name): Score less than 30% ($(configuration_assessment.score))</description>
+  </rule>
+
+  <rule id="19006" level="0">
     <if_sid>19000</if_sid>
     <field name="configuration_assessment.type">^check</field>
     <options>no_full_log</options>
     <description>Configuration Assessment check event.</description>
   </rule>
 
-  <rule id="19003" level="7">
-    <if_sid>19002</if_sid>
+  <rule id="19007" level="7">
+    <if_sid>19006</if_sid>
     <field name="configuration_assessment.check.result">^failed</field>
     <options>no_full_log</options>
-    <description>Check with id $(configuration_assessment.check.id) for policy $(configuration_assessment.policy) failed.</description>
+    <description>$(configuration_assessment.name): $(configuration_assessment.check.title): Failed result.</description>
   </rule>
 
-  <rule id="19004" level="3">
-    <if_sid>19002</if_sid>
+  <rule id="19008" level="3">
+    <if_sid>19006</if_sid>
     <field name="configuration_assessment.check.result">^passed</field>
     <options>no_full_log</options>
-    <description>Check with id $(configuration_assessment.check.id) for policy $(configuration_assessment.policy) passed.</description>
+    <description>$(configuration_assessment.name): $(configuration_assessment.check.title): Passed result.</description>
   </rule>
 
-  <rule id="19005" level="3">
-    <if_sid>19004</if_sid>
+  <rule id="19009" level="3">
+    <if_sid>19008</if_sid>
     <field name="configuration_assessment.check.previous_result">^failed</field>
     <options>no_full_log</options>
-    <description>Check with id $(configuration_assessment.check.id) for policy $(configuration_assessment.policy) has changed to passed.</description>
+    <description>$(configuration_assessment.name): $(configuration_assessment.check.title): Result changed from failed to passed.</description>
   </rule>
 
-  <rule id="19006" level="9">
-    <if_sid>19003</if_sid>
+  <rule id="19010" level="9">
+    <if_sid>19007</if_sid>
     <field name="configuration_assessment.check.previous_result">^passed</field>
     <options>no_full_log</options>
-    <description>Check with id $(configuration_assessment.check.id) for policy $(configuration_assessment.policy) has changed to failed.</description>
+    <description>$(configuration_assessment.name): $(configuration_assessment.check.title): Result changed from passed to failed.</description>
```

Comment:
`$(configuration_assessment.name): $(configuration_assessment.check.title): Result changed from passed to failed.`
->
`$(configuration_assessment.name): $(configuration_assessment.check.title): Status changed from passed to failed.`